src/modules/addItem.js

```diff
@@ -0,0 +1,7 @@
+export default class AddItem {
+  constructor(input) {
+    this.content = input;
+    this.complete = false;
+    this.indexNumber = 0;
+  }
+}
```

Comment:
- [ ] By default, new tasks should have the property `index` should be set set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). Which means that `index` should start from `1`